libconfluo/confluo/container/monolog/monolog_exp2_linear.h

```diff
@@ -432,8 +432,23 @@ class monolog_exp2_linear : public monolog_exp2_linear_base<T, NCONTAINERS, BUCK
   /**
    * Constructs a relaxed monolog implementation
    */
-  monolog_exp2_linear()
-      : tail_(0) {
+  monolog_exp2_linear(size_t count = 0)
```

Comment:
What is the purpose of having a monolog with initialized read-tail? The space may not be allocated for it.

Reply:
No idea, made this change a long time ago. I'll try removing it and see if it breaks anything.